### ISSUE TITLE
Cannot use InputField in PHP 7.1 (TypeError)

### ISSUE THREAD

**Issue by miflinak** [Youshido/GraphQL#157]
1) Run example `03_relay_star_wars`
2) Open it in GraphiQL
3) Get an error:

```
Fatal error:  Uncaught TypeError: Argument 1 passed to Youshido\GraphQL\Introspection\Field\TypesField::collectTypes() must be an instance of Youshido\GraphQL\Type\AbstractType, instance of Youshido\GraphQL\Field\InputField given, called in path\vendor\youshido\graphql\src\Introspection\Traits\TypeCollectorTrait.php on line 86 and defined in path\vendor\youshido\graphql\src\Introspection\Traits\TypeCollectorTrait.php:21
Stack trace:
#0 path\vendor\youshido\graphql\src\Introspection\Traits\TypeCollectorTrait.php(86): Youshido\GraphQL\Introspection\Field\TypesField-&gt;collectTypes(Object(Youshido\GraphQL\Field\InputField))
#1 path\vendor\youshido\graphql\src\Introspection\Traits\TypeCollectorTrait.php(48): Youshido\GraphQL\Introspection\Field\TypesField-&gt;collectFieldsArgsTypes(Object(Youshido\GraphQL\Type\InputObject\InputObjectType))
#2 path\vendor\youshido\g in path\vendor\youshido\graphql\src\Introspection\Traits\TypeCollectorTrait.php on line 21
```

I also tried latest version from git - same error.

P.S. Maybe it's a good idea to do a release with fixed php 7.1 compability (commits fixing it still not in release).

**Comment by Jalle19** [comment]
I agree, we use this library in a project with PHP 7.1 and while it works for us (since we don't use this particular functionality) it would be great to get a released version of the 1.4.x branch that has proper support for it.

**Comment by carere** [comment]
Agree too, we also use this library with PHP 7.1

Since we are on docker, i can switch to 7.0 easily, but it would be nice to release a fix :)

Thx for the bundle. Have a nice day :)

**Comment by viniychuk** [comment]
@Jalle19 @miflinak fixed in 1.5
